website/screens/components/select/code/SelectCodePage.tsx

```diff
@@ -253,63 +244,46 @@ const sections = [
         content: (
           <>
             <DxcText as="p">
-              The state of the component is managed internally by the select.
-              Here is an example of how to use an uncontrolled select the submit
-              event.
+              Alternatively you can let the component manage its state
+              internally. The example below shows how to handle the value of an
+              uncontrolled select in the submit event.
             </DxcText>
-            <Example example={uncontrolled} defaultIsVisible />
-          </>
-        ),
-      },
-      {
-        title: "Searchable",
-        content: (
-          <>
             <DxcText as="p">
-              The <Code>searchable</Code> prop enables the search functionality
-              for filtering the select options.
+              The Select's value is empty by default, but an initial,
```

Comment:
I would be writting `Select` as `select` (with lowercase).

Reply:
I agree, changed!